_cucumber/features/downloads/cdk_downloads.feature

```diff
@@ -0,0 +1,66 @@
+@ignore
+
+Feature: Red Hat Container Development Kit (CDK) download
+
+  In order to build container-based applications,
+  As a registered develpers.redhat.com site visitor,
+  I want to be able to download the redhat CDK.
+
+  @customer
+  @logout
+  @javascript
+  Scenario: Newly registered RHD navigates to CDK Download page and clicks on download, upgrades account, who accepts Redhat T&C's should initiate CDK download.
+    Given I register a new account
+    When I am on the Download Overview page for cdk
+    And I click to download the featured download of "Red Hat Container Development Kit (CDK)"
+    When I upgrade my account and accept the terms and conditions
+    Then I should see the cdk get started page with a confirmation message Thank you for downloading Red Hat Container Development Kit (CDK)
+    And the download should initiate
+
+  @customer
+  @logout
+  @javascript
+  Scenario: Newly registered RHD navigates to CDK Download page and clicks on download, upgrades account, but does NOT accept Redhat T&C's should not initiate CDK download.
```

Comment:
'Newly registered RHD' -> 'Newly registered User'

There are similar occurrences of this elsewhere.